patches/@ephox+wrap-prismjs+1.29.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@ephox/wrap-prismjs/src/main/js/ephox/wrap-prismjs/PrismJS.js b/node_modules/@ephox/wrap-prismjs/src/main/js/ephox/wrap-prismjs/PrismJS.js
+index a0eed9a..60075a8 100644
+--- a/node_modules/@ephox/wrap-prismjs/src/main/js/ephox/wrap-prismjs/PrismJS.js
++++ b/node_modules/@ephox/wrap-prismjs/src/main/js/ephox/wrap-prismjs/PrismJS.js
+@@ -220,7 +220,7 @@ var exports = {}, module = { exports: exports }, global = {};
+ 				setLanguage: function (element, language) {
+ 					// remove all `language-xxxx` classes
+ 					// (this might leave behind a leading space)
+-					element.className = element.className.replace(RegExp(lang, 'gi'), '');
```

Comment:
I don't think I understand what is going on here? Do we own this code? If so, why are we patching it? Can't we just make another version? Or is there an issue with version number syncing with the original prism?

Reply:
Ah thanks for the poke. This is indeed incorrect. I need to take this patch and apply it to prism inside `wrap-prismjs` directly, `contact` already has patching built into it. The application here was just to avoid testing on IE11 myself.